spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -81,12 +81,19 @@
   end
 
   describe '#retrieve_token!' do
+    let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store) }
+    let(:cache) { Rails.cache }
+
+    before do
+      allow(Rails).to receive(:cache).and_return(memory_store)
+      Rails.cache.clear
+    end
```

Comment:
should we switch all of the test env to use a memory store? and maybe a `before(:all)` that resets it before every test?

https://github.com/18F/identity-idp/blob/3b272cc15048c65a5c40ff3e6f1a6a0a826ef88e/config/environments/test.rb#L10